service/src/main/java/org/apache/griffin/core/job/entity/JobInstanceBean.java

```diff
@@ -85,6 +85,8 @@ Licensed to the Apache Software Foundation (ASF) under one
     @JsonIgnore
     private AbstractJob job;
 
+    private String triggerName;
```

Comment:
this will change database schema, isn't it?
will that schema change be applied automatically to database table?
if not, how to migrate database between versions?

Reply:
This is schema change will be applied automatically